test/services/postcard.spec.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Sinon   = require('sinon');
+
+var Config = require('../../config');
+
+require('angular-mocks');
+
+describe('postcard service', function () {
+
+  var $q;
+  var $rootScope;
+  var API;
+  var Postcard;
+
+  beforeEach(Angular.mock.inject(function ($injector) {
+    $q         = $injector.get('$q');
+    $rootScope = $injector.get('$rootScope');
+    API        = $injector.get('API');
+    Postcard   = $injector.get('Postcard');
+  }));
+
+  describe('create', function () {
+
+    it('calls the correct endpoint and with correct params', function () {
+      var payload = { id: 'psc_id' };
+
+      Sinon.stub(API, 'post').returns();
```

Comment:
Oh yeah, that is what it said before and I think you were showing me Sinon stub and other functions when I asked you to explain it and I forgot to change it back. It makes sense, I'll fix it!